float32 aat_pitch
float32 aat_roll
float32 aat_yaw
float32 aat_ax
float32 aat_ay
float32 aat_az

int32 aat_alt
int32 aat_temp

string textos
